### PR TITLE
feat: similar news command & slash command handler and deployer

### PR DESCRIPTION
**Pull request by DevShaded**
## Pull Request Description

I have added a slash command deployer so the bot can have slash commands `createApplicationCommands.ts`. 
I have also added a slash command handler so we can handle slash commands that are created for the bot. 
I have made a slash command that gets similar news from the VG API. 

## How Has This Been Tested?

I have tested the project locally on my own computer. Everything should be working fine.

### CODE REVIEW

File: src/commands/news/similarNews.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    .setDescription('Get similar news')
+    .addStringOption(option => (
+        option.setName('search-similarity')
+            .setDescription('Enter search similarity')
+            .addChoices(
+                {
+                    name: 'Vg articles (recommended)',
+                    value: 'vg_articles_ada'
+                })
+            .setRequired(true)))
```
Comment (jgorset) [inline src/commands/news/similarNews.ts]: If this is required and there's only one choice, I guess we don't need it

File: src/commands/news/similarNews.ts [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    .addStringOption(option => (
+        option.setName('articletitle')
+            .setDescription('Enter article title')
```
Comment (jgorset) [inline src/commands/news/similarNews.ts]: This is more like "searchterm", no?

**Review by jgorset — CHANGES_REQUESTED** [review]

File: src/commands/news/similarNews.ts (hunk 2 above)
Comment (DevShaded) [reply src/commands/news/similarNews.ts]: yes

**Review by jgorset — APPROVED** [review]
Very nice ✨ 🍰 ✨

**Pull request merged**